DependencyInjection/Configuration.php

```diff
@@ -65,41 +64,19 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
                 ->arrayNode('document_tree')
-                    ->useAttributeAsKey('class')
-                    ->prototype('array')
-                        ->fixXmlConfig('valid_child', 'valid_children')
-                        ->children()
-                            ->arrayNode('valid_children')
-                                ->prototype('scalar')->end()
-                                ->info('class names of valid children, manage tree operations for them and hide other children')
-                            ->end()
-                            ->scalarNode('image')
-                                ->defaultValue('')
-                            ->end()
-                        ->end()
-                    ->end()
-                 ->end()
-
-                 ->scalarNode('document_tree_repository')->defaultValue('default')->end()
-
-                ->arrayNode('document_tree_defaults')
-                    ->prototype('scalar')->end()
-                ->end()
-
-                ->arrayNode('document_tree_options')
-                    ->addDefaultsIfNotSet()
+                    ->canBeEnabled()
                     ->children()
-                        ->integerNode('depth')
-                            ->defaultValue(1)
-                            ->info('Depth to which to fetch tree children when rendering the initial tree')
+                        ->scalarNode('repository_name')
+                            ->defaultValue('default')
```

Comment:
The tree does not ask the manager registry to fetch the repositor. The tree creates an api url lik `/de/admin/default/cms/content/content-1` So the repository name is a string in this bundle only. 
I am not sure if i am able to set the default name in case of `null` inside the bundles extension class. Or the admin should do it. I will try the last idea first.